dev-libs/tvision/tvision-2.2.1.4.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Text User Interface that implements the well known CUA widgets"
+HOMEPAGE="http://tvision.sourceforge.net/"
+MY_PVR=${PVR:0:5}-${PVR:6}
+SRC_URI="mirror://sourceforge/tvision/rhtvision_${MY_PVR}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+X debug gpm"
+
+DOCS=( readme.txt THANKS TODO )
+HTML_DOCS=( www-site/. )
+
+S=${WORKDIR}/${PN}
+
+# installed lib links to those
+RDEPEND="sys-libs/ncurses:0=
+	X? ( x11-libs/libX11
```

Comment:
the first line as well (the indentation looks weird otherwise)

Reply:
Done
I also changed the ${PN}-${PVR} in src_install rm command to ${P}, it's no longer necessary.